### PR TITLE
Feature windows build script

### PR DESCRIPTION
**Pull request by juliuskreutz**
Added build script for windows, that downloads the [npcap sdk](https://npcap.com/#download) and set the `LIB` environment variable to the right directory.

### CODE REVIEW

**Comment by IceDynamix** [comment]
if i read this correctly, it performs the entire thing with download and everything on every build? this shouldn't happen. check out the chapter on [change detection for build scripts](https://doc.rust-lang.org/cargo/reference/build-scripts.html#change-detection) in the cargo reference to get a better feel for it. 

the most simple solution would probably be emitting `cargo::rerun-if-changed=build.rs`, so it only ever runs once (as described in the chapter). i'm not 100% sure though if this is the best solution though, since it's just gut feeling typing from my phone. you'll have to look a bit more into this to see if there might be something more appropriate.

File: build.rs [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+                if !p.exists() {
+                    std::fs::create_dir_all(p)?;
+                }
+            }
+            let mut outfile = std::fs::File::create(&outpath)?;
+            std::io::copy(&mut file, &mut outfile)?;
+        }
+    }
+
+    let arch_dir = if cfg!(target_arch = "x86_64") {
```
Comment (IceDynamix) [inline build.rs:45]: i think using the same string as the target arch for the dir name is better

File: build.rs [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+    let mut bytes = Vec::new();
+    ureq::get("https://npcap.com/dist/npcap-sdk-1.13.zip")
+        .call()?
+        .into_reader()
+        .read_to_end(&mut bytes)?;
+    let cursor = std::io::Cursor::new(bytes);
+    let mut archive = zip::ZipArchive::new(cursor)?;
+
+    for i in 0..archive.len() {
```
Comment (IceDynamix) [inline build.rs:20]: isn't using `archive.by_name("Lib")` more appropriate?

**Comment by juliuskreutz** [comment]
Yeah after coming across an issue with windows-aarch64 I also think, that using a build script for this is probably not the best idea.

**Pull request closed without merging**